content/ri/memory-optimizations.md

```diff
@@ -67,7 +67,7 @@ Looking at the difference you can clearly point out that lot of memory is still
 You can follow one of these three steps to reclaim the memory:
 
 1. Restart your redis-server
-1. Increase memorysamples in redis conf. (default is 5, max is 10) so that expired keys are reclaimed faster.
+1. Increase `maxmemory-samples` in redis.conf (default is 5, max is 10) so that the expired keys are reclaimed faster.
```

Comment:
`max is 10` is wrong, 10 only means better accuracy, but will consume more cpu, its max value is int_max.

Reply:
Here is another `memorysamples`.
```
If we increase the memorysamples config, it expires the keys faster, but it costs more CPU cycles, which increases latency of commands. Secondly, increasing the expiry of keys helps but that requires significant changes to application logic.
```

Reply:
@sundb, should we say "To reclaim the expired keys faster, increase `memory-samples` in redis.conf (default is 5)."

Reply:
seems good to me.